### PR TITLE
iko-yo.net

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [x] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 https://iko-yo.net/articles/2826

### Add your comment and screenshots

`iko-yo.net##.c-ad-information` should move to JPF? or stay mobile filter?

<details><summary>screenshot - mobile</summary>

</details><br/>

<details><summary>screenshot - PC</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: JapaneseFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -101,6 +101,8 @@ maidonanews.jp#$?#.module-list-articles__item[style="padding:0px"] { remove: tru
 maidonanews.jp#?#.module-ad:has(> div.module-ad__item > div[id^="mn"])
 games.app-liv.jp#?#.special_newarticle_list > li:has(> div.adsbygoogle_area)
 myoji-yurai.net#?#.spSideNone:has(> li > div.app_icon)
+iko-yo.net#?#div[class*="c-ad-"]:has(> div[id^="div-gpt-"])
+iko-yo.net#?#ul[class^="c-side-list"] > li.c-ad--tieup:has(> div[id^="div-gpt-"])
```
Comment (Yuki2718) [inline JapaneseFilter/sections/css_extended.txt:105]: No need to use extended syntax:
```
iko-yo.net##.c-ad--h250
iko-yo.net##.c-ad--tieup
iko-yo.net##.c-ad_container
```

File: MobileFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -7,6 +7,7 @@
 !
 !
 realtime-chart.info#?#.tb-n tr[bgcolor="#fff"]:has(span:contains(PR))
+iko-yo.net#?#ul[class^="c-list"] > li.c-ad--tieup:has(> div[id^="div-gpt-"])
```
Comment (Yuki2718) [inline MobileFilter/sections/css_extended.txt:10]: Same

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

**Comment by Yuki2718** [comment]
> `iko-yo.net##.c-ad-information` should move to JPF? or stay mobile filter?

It's on PC site too so move to JPF.

File: JapaneseFilter/sections/css_extended.txt (hunk 1 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/css_extended.txt:105]: ok

File: MobileFilter/sections/css_extended.txt (hunk 2 above)
Comment (SKEIDs) [reply MobileFilter/sections/css_extended.txt]: ok

File: JapaneseFilter/sections/specific.txt [hunk 3]
```diff
@@ -907,7 +907,11 @@ news-channel.doorblog.jp##.headerBnr
 ||0matome.com/img/pr_k-jz.png
 ||fc2.com/distribution/preroll/
 keibana.com##.ad-g
-keibana.com##.l-header__ad
+iko-yo.net##.c-ad-information
+iko-yo.net##.c-ad--h250
+iko-yo.net##.c-ad--tieup
+iko-yo.net##.c-ad_container
+iko-yo.net,keibana.com##.l-header__ad
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:914]: Please also add `iko-yo.net##.c-ad--w468`, it's in the top page `https://iko-yo.net/`.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt (hunk 3 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:914]: ok.

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**